common/app/common/NewNavigation.scala

```diff
@@ -119,86 +70,114 @@ object NewNavigation {
   case object News extends EditionalisedNavigationSection {
     val name = "news"
 
-    val uk = List(headlines, ukNews, world, politics, environment, business, tech, science, money)
-    val au = List(headlines, australiaNews, world, auPolitics, environment, economy, auImmigration, indigenousAustralia, media, tech)
-    val us = List(headlines, usNews, usPolitics, world, environment, business, tech, science, money)
-    val int = List(headlines, world, ukNews, environment, business, tech, science, cities, globalDevelopment)
+    val uk = NavLinkLists(
+      List(headlines, ukNews, world, politics, science, business),
+      List(tech, environment, money)
+    )
+    val au = NavLinkLists(
+      List(headlines, australiaNews, world, auPolitics, auImmigration),
+      List(indigenousAustralia, economy, tech, environment, media)
+    )
+    val us = NavLinkLists(
+      List(headlines, usNews, world, science, usPolitics, business),
+      List(environment, money, tech)
+    )
+    val int = NavLinkLists(
+      List(headlines, world, ukNews, science, cities, globalDevelopment),
+      List(tech, business, environment)
+    )
   }
 
   case object Opinion extends EditionalisedNavigationSection {
     val name = "opinion"
 
-    val uk = List(
-      opinion,
-      NavLink("Polly Toynbee", "/profile/pollytoynbee"),
-      NavLink("Owen Jones", "/profile/owen-jones"),
-      NavLink("Marina Hyde", "/profile/marinahyde"),
-      NavLink("George Monbiot", "/profile/georgemonbiot"),
-      NavLink("Gary Younge", "/profile/garyyounge"),
-      NavLink("Nick Cohen", "/profile/nickcohen"),
-      columnists,
-      theGuardianView,
-      cartoons
+    val uk = NavLinkLists(
+      List(opinion, NavLink("Polly Toynbee", "/profile/pollytoynbee"), NavLink("Owen Jones", "/profile/owen-jones"), NavLink("Marina Hyde", "/profile/marinahyde")),
+      List(NavLink("George Monbiot", "/profile/georgemonbiot"), NavLink("Gary Younge", "/profile/garyyounge"), NavLink("Nick Cohen", "/profile/nickcohen"), columnists, theGuardianView, cartoons)
     )
 
-    val au = List(
-      opinion,
-      NavLink("first dog on the moon", "/profile/first-dog-on-the-moon"),
-      NavLink("Katharine Murphy", "/profile/katharine-murphy"),
-      NavLink("Kristina Keneally", "/profile/kristina-keneally"),
-      NavLink("Richard Ackland", "/profile/richard-ackland"),
-      NavLink("Van Badham", "/profile/van-badham"),
-      NavLink("Lenore Taylor", "/profile/lenore-taylor"),
-      NavLink("Jason Wilson", "/profile/wilson-jason"),
-      NavLink("Brigid Delaney", "/profile/brigiddelaney"),
-      columnists
+    val au = NavLinkLists(
```

Comment:
Just consistency, but looking at it again this morning I agree with you! I will change all the opinion to be multi line again.

Thanks :)